src/catalog_api.cpp

```diff
@@ -135,10 +135,12 @@ IRCAPITableCredentials IRCAPI::GetTableCredentials(ClientContext &context, IRCat
 
 	case_insensitive_map_t<Value> config_options;
 	auto *config_val = yyjson_obj_get(root, "config");
-	if (config_val && catalog_credentials) {
+	if (catalog_credentials) {
```

Comment:
@Tishj this change I'm not 100% sure about. The issue is that if an S3 secret is created for s3tables, we need to create a temporary secret for the storage (with scope etc.) that also has the region which is inferred by the warehouse. 
So in this case, `config` is empty, but we have catalog credentials, and we want to use those

Reply:
I think this was only following the old behavior, I'm fine with this change. 👍